client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -235,6 +241,31 @@ class NotificationSubscriptions extends Component {
 							</FormLabel>
 						</FormFieldset>
 
+						{ isAutomattician && (
+							<FormFieldset>
+								<FormLegend>
+									{ this.props.translate(
+										'Auto-follow P2 posts upon commenting (Automatticians only)'
```

Comment:
Actually, we might want to skip localizing the strings if the feature is really "automatticians only". Why pay for additional strings when these are for internal use 😉

Reply:
Agreed! Addressed in 0266a9a947